src/sql-parser/src/parser.rs

```diff
@@ -2200,13 +2200,41 @@ impl<'a> Parser<'a> {
         Ok(CsrConnectionProtobuf { connection, seed })
     }
 
+    fn parse_source_error_policy_option(&mut self) -> Result<SourceErrorPolicy, ParserError> {
+        if self.parse_keyword(INLINE) {
+            Ok(SourceErrorPolicy::Inline)
+        } else if self.parse_keyword(PROPAGATE) {
+            Ok(SourceErrorPolicy::Propagate)
+        } else {
+            self.expected(self.peek_pos(), "INLINE, or PROPAGATE", self.peek_token())
+        }
```

Comment:
You can probably replace this with something like

```Rust
match self.expect_one_of_keywords(&[INLINE, PROPAGATE]) {
  INLINE => Ok(SourceErrorPolicy::Inline),
  PROPAGATE => Ok(SourceErrorPolicy::Propagate),
  _ => unreachable!(),
```

That helps with generating the error message and makes sure it's consistent with other errors.